### ISSUE TITLE
iOS builds are failing with an error `Duplicate interface definition for class 'RCTConvert'`

### ISSUE THREAD

**Issue by justblender** [Instabug/Instabug-React-Native#660]
Instabug SDK fails an iOS build when used in a project running React Native 0.65.2 with an error `Duplicate interface definition for class 'RCTConvert'`, which leads to the following files: 

1. [RCTConvert+InstabugEnums.h (line 10)](https://github.com/Instabug/Instabug-React-Native/blob/master/ios/RNInstabug/RCTConvert%2BInstabugEnums.h)
2. [RCTConvert+InstabugEnums.m (line 9)](https://github.com/Instabug/Instabug-React-Native/blob/master/ios/RNInstabug/RCTConvert%2BInstabugEnums.m)

I'm guessing that some earlier release have changed the way how React Native files need to be included in Objective-C code and now require prepending `React/` to all includes, but I'm definitely not sure if this is the case. I'd love to hear a more detailed explanation to this.

For now, I figured out that swapping conditions around and favoring namespaced `RCTConvert` header file results in a successful build:
```diff
-#if __has_include("RCTConvert.h")
-  #import "RCTConvert.h"
-#else
+#if __has_include("React/RCTConvert.h")
   #import <React/RCTConvert.h>
+#else
+  #import "RCTConvert.h"
 #endif
```

I'd like to propose a PR for this, but I'm not sure if this is a hacky solution or not.

### Steps to Reproduce the Problem
1. Initialize an empty React Native project (version 0.65.2);
2. Install the latest `instabug-reactnative` package;
3. Follow installation steps for this package;
4. Build the project for iOS devices. 

### Expected Behavior
Expecting builds to succeed

### Actual Behavior
Builds are failing

### Instabug integration code
Unnecessary for this issue

### SDK Version
SDK version: `10.11.0`

### React Native, iOS and Android Versions
React Native version: `0.65.2`
iOS version: `15.2`

### Device Model
Running on an iOS simulator on M1 MacBook Air (Late 2020)

**Comment by ahmed1hisham** [comment]
Hello @justblender,
Thank you for the thorough details. I have checked the issue and I can confirm that the fix you mention is a valid one. However, this fix is already introduced in our SDK in the same file you specified. Please check it out [here](https://github.com/Instabug/Instabug-React-Native/blob/dd6075a7483fad76c7b0317ddacf69d118208b44/ios/RNInstabug/RCTConvert%2BInstabugEnums.h#L9).

I will close the issue for now, please feel free to re-open it if you have any other concerns 🙏

**Comment by justblender** [comment]
> Hello @justblender, Thank you for the thorough details. I have checked the issue and I can confirm that the fix you mention is a valid one. However, this fix is already introduced in our SDK in the same file you specified. Please check it out [here](https://github.com/Instabug/Instabug-React-Native/blob/dd6075a7483fad76c7b0317ddacf69d118208b44/ios/RNInstabug/RCTConvert%2BInstabugEnums.h#L9).

Hello @ahmed1hisham, thank you for looking into my issue. Unfortunately, I don’t see any changes in a linked file, the latest commit for it was made in 2018. Maybe the fix was applied in a different branch?

**Comment by ahmed1hisham** [comment]
@justblender 
Oh, I misread the introduced fix, sorry about that. However, I double checked the issue and after following the reproduction steps, it was not reproducible from our end using the latest SDK version 10.11.0  and RN version 0.65.2 .

This SDK version includes the following condition (which suffices in this case):
```
#if __has_include("RCTConvert.h")
  #import "RCTConvert.h"
#else
  #import <React/RCTConvert.h>
#endif
```
Can you please retry your steps on a newly created app and make sure that you clean the build folder? 🙏 
Please re-open the issue if it is still reproducible so that we can investigate the cause of output difference between your side and ours.
Would also be great if you can provide a sample app where this is reproducible.

**Comment by justblender** [comment]
@ahmed1hisham, yeah, just tried to reproduce the issue and builds succeed for me too. Strange. 

Still, I've decided to investigate this issue since it still affects my real project and I've found this [commit](https://github.com/facebook/react-native/commit/e1577df1fd70049ce7f288f91f6e2b18d512ff4d), which moved all React Native header files to `React` namespace. 

I think we should respect this change and favour namespaced imports, especially since it shouldn't break existing projects. LMK if I should send a PR over your way.
